main.cpp

```diff
@@ -546,13 +436,128 @@ MOVE THEM to the space below this block comment and put them in numerical order
 */
 
 
+/*
 
+Thing 1) TV
+5 properties:
+    1) broadcasted channel (int)
+    2) volume level (int)
+    3) amount of display ports (int)
+    4) internet connection (bool)
+    5) amount of smartTV apps (number)
```

Comment:
`number` is not a C++ type.  pick a primitive (`int, float, bool, char, double`)
- [ ] revise